homeassistant/components/monarch_money/coordinator.py

```diff
@@ -63,9 +63,15 @@ async def _async_setup(self) -> None:
     async def _async_update_data(self) -> MonarchData:
         """Fetch data for all accounts."""
 
+        now = datetime.now()
+        start_of_year = now.replace(month=1, day=1).strftime("%Y-%m-%d")
+        end_of_year = now.replace(month=12, day=31).strftime("%Y-%m-%d")
```

Comment:
This can be simplified:

```suggestion
        now = datetime.now()
        start_of_year = f"{now.year}-01-01"
        end_of_year = f"{now.year}-12-31"
```

You could even add those f-string directly into the method call down below.